cmd/nem/main.go

```diff
@@ -102,6 +106,21 @@ func main() {
 						},
 					},
 				},
+				{
+					Name:   "info",
+					Usage:  "extract info from account",
+					Action: info,
+					Flags: []cli.Flag{
+						cli.BoolFlag{
+							Name:  "address-only, a",
```

Comment:
Shorthand is good only for most popular options, it's not obvious what is it - `nem account info -a`.